barretenberg/cpp/src/barretenberg/proof_system/circuit_builder/eccvm/eccvm_circuit_builder.test.cpp

```diff
@@ -240,32 +255,36 @@ TYPED_TEST(ECCVMCircuitBuilderTests, MSM)
     static constexpr size_t max_num_msms = 9;
     auto generators = G1::derive_generators("test generators", max_num_msms);
 
-    const auto try_msms = [&](const size_t num_msms, auto& circuit) {
+    const auto try_msms = [&](const size_t num_msms, auto& op_queue) {
```

Comment:
why is it called "try_msms"?

Reply:
good question, i guess it could be perform_msms

Reply:
or compute